backend/api_calls.py

```diff
@@ -760,6 +760,40 @@ def RemovePlayerGroupMappings(firebase, group, player):
       'Removed player %s from %d missions.' % (player, len(missions))]
 
 
+def AutoUpdatePlayerGroups(firebase, player, new_player=False):
```

Comment:
nit: I would prefer this to be called player_id instead of because it can be confusing when this is a player object or a single string player_id.